### ISSUE TITLE
setup a heroku server

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-18#3]

**Comment by JorgeRodr** [comment]
@atherdon I'm gonna use this tutorial

https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction

Correct me if I'm wrong ;)

**Comment by atherdon** [comment]
it's a good article. i started from it too.
main goal is to have a server online - so sometimes, in order to answer your questions - i don't need to setup a local version

**Comment by atherdon** [comment]
did you setup a heroku server? can i close this task?

**Comment by JorgeRodr** [comment]
@atherdon I have installed the heroku-cli and logged in with a student accound I have created, but I didn't create and deploy the app, should I?

**Comment by atherdon** [comment]
yeah, please deploy it, this will be important skill later. when you'll deploy it - please push Heroku link to readme and close this taks
